paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -254,27 +254,6 @@ set(BERT_INSTALL_DIR "${INFERENCE_DEMO_INSTALL_DIR}/bert_emb128")
 download_model_and_data(${BERT_INSTALL_DIR} "bert_emb128_model.tar.gz" "bert_data_len20.txt.tar.gz")
 inference_analysis_api_test(test_analyzer_bert ${BERT_INSTALL_DIR} analyzer_bert_tester.cc)
```

Comment:
Do `anakin_rnn1_tester.cc` and `anakin_mobilenet_tester.cc` be removed at the same time?

Reply:
ok, I removed them.